multisrc/overrides/madara/chunchunmarutranslations/src/ChunchunmaruTranslations.kt

```diff
@@ -0,0 +1,35 @@
+package eu.kanade.tachiyomi.extension.es.chunchunmarutranslations
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.SChapter
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ChunchunmaruTranslations : Madara(
+    "Chunchunmaru Translations",
+    "https://chunchunmarutl.com",
+    "es",
+    SimpleDateFormat("dd 'de' MMMMM 'de' yyyy", Locale("es"))
+) {
+
+    override fun popularMangaRequest(page: Int): Request = GET("$baseUrl/manga/page/$page/?m_orderby=views", headers)
+    override fun latestUpdatesRequest(page: Int): Request = GET("$baseUrl/manga/page/$page/?m_orderby=latest", headers)
+
+    override fun chapterFromElement(element: Element): SChapter {
+        val chapter = super.chapterFromElement(element)
+        // We are assuming that the number will be always in the name
+        chapter.chapter_number = chapter.name.split(" ")[1].toFloat()
+        return chapter
+    }
```

Comment:
This will easily break if they change the pattern or include an invalid number.

```suggestion
    override fun chapterFromElement(element: Element): SChapter {
        return super.chapterFromElement(element).apply {
            chapter_number = name.split(" ").getOrNull(1)?.toFloatOrNull() ?: -1f
        }
    }
```